tests/test_helpers/test_acn/test_uri.py

```diff
@@ -30,3 +30,12 @@ def test_uri():
     assert uri.host == "localhost"
     assert uri.port == 9000
     Uri()
+
+
+def test_uri2():
+    """Test the uri."""
+    Uri(host="127.0.0.1")
+    uri = Uri(host="127.0.0.1", port=10000)
+    assert str(uri) == "127.0.0.1:10000"
+    assert uri.host == "127.0.0.1"
+    assert uri.port == 10000
```

Comment:
https://github.com/valory-xyz/open-aea/blob/6395b608560d6e7f2d1ed043087dd4c375cb3b39/tests/test_packages/test_connections/test_p2p_libp2p_client/test_communication.py#L553-L557

this resided in libp2p tests, which is not the right place, instead of removing it I moved it here